### PR TITLE
Rework pokestop popup

### PR DESCRIPTION
**Pull request by na-ji**
## Description
Replace `Pokéstop` with the pokestop name in pokestop popup
Add pokestop image to pokestop popup
Add full quest task to pokestop popup

## Screenshots (if appropriate):
image
image
image

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- My change requires a change to the documentation.
- I have updated the documentation accordingly.

### CODE REVIEW

**Comment by na-ji** [comment]
I fixed a little alignment problem when the task is too long. 
What do you thing @cecpk ?

File: static/js/map.js [hunk 1]
```diff
@@ -997,82 +997,85 @@ function gymLabel(gym, includeMembers = true) {
...
             </div>
+            <div class="pokestop-quest__task">${quest['quest_task']}</div>
           </div>`
     }
 
-    return str
+    return `
+      <div>
+        <div class='pokestop ${luredClass}'>
+          ${pokestop.name}
```
Comment (fosJoddie) [inline static/js/map.js]: Missing fallback to "pokestop" for any stop that has not been scanned (new stops, or stops in areas not quest scanned)

Comment (na-ji) [reply static/js/map.js]: Thanks! How about now?

Comment (na-ji) [reply static/js/map.js]: image
We fallback to the old layout when the stop does not have a name nor an image. So I think we're good to go and I will merge the PR

**Pull request merged**